### DISCUSSION TITLE
Zbbridge: Accessing attributes on TS130F curtain module (calibration)

### DISCUSSION THREAD

**Discussion by peturdainn** [arendst/Tasmota#15651] [Q&A] [answered]
Failing to access any device-specific attributes using Tasmota on Sonoff Zbbridge.
Here's what I already got: (my device short address = 0x0FEC)
Doc & references at the bottom

Reading basic cluster works, for example model identifier:
CMD: ZbSend {"Device":"0x0FEC","Endpoint":1,"Cluster":"0x0000","Read":"0005"} 
MQT: Zbridge1/RESULT = {"ZbSend":"Done"}
MQT: Zbridge1/SENSOR = {"ZbReceived":{"0x0FEC":{"Device":"0x0FEC","ModelId":"TS130F","Endpoint":1,"LinkQuality":131}}}

but the Window Covering cluster (0x0102 or 258) doesn't respond at all (for example: currentPositionLiftPercentage)

CMD: ZbSend {"Device":"0x0FEC","Endpoint":1,"Cluster":"0x0102","Read":"0008"}

I tried using the raw syntax but either it fails too or I'm confused how it deals with commands and attributes

So what is going on? From the zigbee2mqtt issue I understand they got it working, their details match the spec, but it's not working from Tasmota Zbbridge.

Thanks!

Useful links:

list of clusters, commands and and attributes
https://zigbeealliance.org/wp-content/uploads/2019/12/07-5123-06-zigbee-cluster-library-specification.pdf

details of zigbee2mqtt implementation for the  attributes of this module:
https://github.com/Koenkk/zigbee2mqtt/issues/7128

**Comment by alsak0de** [comment, marked as answer]
take a look at this... https://github.com/arendst/Tasmota/discussions/13003

**Reply by peturdainn** [reply]
thanks a lot, that's it!
